pkg/module/metrics/latency.go

```diff
@@ -125,10 +124,10 @@ func (lm *LatencyMetrics) Init(metricName string) {
 		if reason == ttlcache.EvictionReasonExpired {
 			// Didn't get the corresponding packet.
 			lm.l.Debug("Evicted item", zap.Any("item", item))
-			if lm.noResponseMetric != nil {
-				lm.noResponseMetric.WithLabelValues("no_response").Inc()
-				lm.l.Debug("Incremented no response metric", zap.Any("metric", lm.noResponseMetric))
-			}
+			// if lm.noResponseMetric != nil {
+			// 	lm.noResponseMetric.WithLabelValues("no_response").Inc()
+			// 	lm.l.Debug("Incremented no response metric", zap.Any("metric", lm.noResponseMetric))
+			// }
```

Comment:
We still want this metric, what we don't want is the [label](https://github.com/microsoft/retina/blob/debc18823a6535deb9c3d4f6d18dd7aca3c2916c/pkg/module/metrics/latency.go#L110) `count` since it's redundant. 

So, instead of 
```
adv_node_apiserver_no_response{count="no_response"} 10
```
we want
```
adv_node_apiserver_no_response{} 10
```